contrib/lang/scala/packages.el

```diff
@@ -47,6 +47,66 @@ which require an initialization must be listed explicitly in the list.")
         (kbd "n") 'forward-button
         (kbd "N") 'backward-button)
 
+      (evil-leader/set-key-for-mode 'scala-mode
+        "mri"     'ensime-refactor-inline-local
+        "mrl"     'ensime-refactor-extract-local
+        "mrm"     'ensime-refactor-extract-method
+        "mro"     'ensime-refactor-organize-imports
+        "mrr"     'ensime-refactor-rename
+        "mrt"     'ensime-import-type-at-point
+
+        "mbS"     'ensime-stacktrace-switch
+        "mbT"     'ensime-sbt-do-test
+        "mbc"     'ensime-sbt-do-compile
+        "mbn"     'ensime-sbt-do-clean
+        "mbo"     'ensime-sbt-do-test-only
+        "mbp"     'ensime-sbt-do-package
+        "mbr"     'ensime-sbt-do-run
+        "mbs"     'ensime-sbt-switch
+        "mbt"     'ensime-sbt-do-test-quick
+
+        "mda"     'ensime-db-clear-all-breaks
+        "mdb"     'ensime-db-set-break
+        "mdc"     'ensime-db-continue
+        "mdd"     'ensime-db-start
+        "mdi"     'ensime-db-inspect-value-at-point
+        "mdl"     'ensime-db-list-locals
+        "mdn"     'ensime-db-next
+        "mdo"     'ensime-db-step-out
+        "mdq"     'ensime-db-quit
+        "mdr"     'ensime-db-run
+        "mds"     'ensime-db-step
+        "mdt"     'ensime-db-backtrace
+        "mdu"     'ensime-db-clear-break
+
+        "mti"     'ensime-goto-impl
+        "mtt"     'ensime-goto-test
+
```

Comment:
So `ensime-goto-{test,impl}` is used to jump between test and implementation files, so it's not a mismatch with spacemacs conventions if we want to keep them this way.